mitodl/ocw-studio
# Add identifier for external links to import_ocw_course_sites

#### Pre-Flight checklist


#### What are the relevant tickets?
Fixes #502 

#### What's this PR do?
Updates `import_ocw2hugo_menu` to create an identifier string for menu entries. Looking at the hugo docs, I don't think `identifier` is strictly necessary for every menu item, but it's used in our frontend code to keep track of menu items so it seems simpler to just generate one.

#### How should this be manually tested?
 - Run `./manage.py import_ocw_course_sites -b ocw-to-hugo-output-qa --filter res-7-003-brave-new-planet-fall-2020`
 - View http://localhost:8043/sites/res-7-003-brave-new-planet-fall-2020/type/menu/. You should see one menu item
